build.sbt

```diff
@@ -1,7 +1,11 @@
+val currentScalaVersion = "2.11.11"
+
 organization in ThisBuild := "com.thoughtworks.akka-http-webjars"
 
-crossScalaVersions in ThisBuild := Seq("2.10.6", "2.11.8")
+scalaVersion in ThisBuild := currentScalaVersion
```

Comment:
Don't add this setting. This setting prevent this project from being referenced as a `RootProject` by another sbt project.

Reply:
Yes, but then it tries to compile for Scala 2.10 which is something that I don't want (akka-http is not published for Scala 2.10)

Reply:
I put `scalaVersion in ThisBuild := "2.11.11"` setting in a `local.sbt` in case of being added to git repository.